.deepsource.toml

```
@@ -39,11 +39,7 @@ enabled = true
   dockerfile_paths = [
     "dockers/agent/ngt/Dockerfile",
     "dockers/discoverer/k8s/Dockerfile",
-    "dockers/gateway/vald/Dockerfile",
-    "dockers/manager/backup/mysql/Dockerfile",
-    "dockers/manager/backup/cassandra/Dockerfile",
-    "dockers/meta/cassandra/Dockerfile",
-    "dockers/meta/redis/Dockerfile"
+    "dockers/gateway/vald/Dockerfile"
```

```suggestion
    "dockers/agent/sidecar/Dockerfile",
    "dockers/gateway/lb/Dockerfile",
    "dockers/gateway/filter/Dockerfile",
    "dockers/manager/index/Dockerfile",
    "dockers/operator/helm/Dockerfile"
```

---

btw should we include other docker files?
```suggestion
    "dockers/agent/sidecar/Dockerfile",
    "dockers/gateway/lb/Dockerfile",
    "dockers/gateway/filter/Dockerfile",
    "dockers/manager/index/Dockerfile",
    "dockers/operator/helm/Dockerfile",
    "dockers/ci/base/Dockerfile",
    "dockers/dev/Dockerfile",
    "dockers/tools/cli/loadtest/Dockerfile"
```

---

I applied my suggestion, if you think other is not needed please let me know